app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupUnlockViewModel.kt

```diff
@@ -38,6 +39,8 @@ class SetupUnlockViewModel @Inject constructor(
             userId = userId,
             cipher = biometricsEncryptionManager.getOrCreateCipher(userId = userId),
         )
+        // whether or not the user has completed the initial setup prior to this.
+        val isInitialSetup = savedStateHandle[SETUP_UNLOCK_INITIAL_SETUP_ARG] ?: false
```

Comment:
Can we use the arguments pattern?

Normally we define an arguments model in the `SetupUnlockNavigation`

Reply:
much better